### PR TITLE
AWS GuardDuty - handle datetime in finding obj serialization 

### PR DESCRIPTION
**Pull request by Itay4**
## Status
- [x] Ready

## Description
`parse_incident_from_finding` fails on `serialization` a finding object with `datetime` in it

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Tests

### CODE REVIEW

**Comment by Itay4** [comment]
TDD: see test failure in https://app.circleci.com/pipelines/github/demisto/content/108711/workflows/c576651f-f7a9-4946-a0df-a81a1a91f429/jobs/414213

File: Packs/AWS-GuardDuty/Integrations/AWSGuardDuty/AWSGuardDuty_test.py [hunk 1]
```diff
@@ -33,3 +35,274 @@ def test_get_members(mocker):
...
+
+    When:
+    - Parsing finding to incident
+
+    Then:
+    - Ensure finding is parsed as expected
+    """
+    title = 'title'
+    desc = 'desc'
+    finding = {
```
Comment (yaakovi) [inline Packs/AWS-GuardDuty/Integrations/AWSGuardDuty/AWSGuardDuty_test.py:53]: can we minimize the response to relevant fields only? should be focused on the relevant fields.
also, add an assert to check the date value.

**Review by yaakovi — APPROVED** [review]

File: Packs/AWS-GuardDuty/Integrations/AWSGuardDuty/AWSGuardDuty_test.py (hunk 1 above)
Comment (ShahafBenYakir) [reply Packs/AWS-GuardDuty/Integrations/AWSGuardDuty/AWSGuardDuty_test.py:53]: Yes

Comment (Itay4) [reply Packs/AWS-GuardDuty/Integrations/AWSGuardDuty/AWSGuardDuty_test.py:53]: keeping the whole finding for testing the the func better
adding assertion of the date parse

**Comment by Itay4** [comment]
not bumping docker image due to insufficient test coverage
merging

**Pull request merged**